memebot/memebot.py

```diff
@@ -200,6 +206,19 @@ async def queue_remove(command: Command, channel: discord.TextChannel, player_co
         await queue_display(command.arg, channel, player_config)
     return queue_remove
 
+
+def create_now_playing_executor(player_config_provider: PlayerConfigProvider) -> CommandExecutor:
+    @filter_player_commands(player_config_provider=player_config_provider, voice_channel_restricted=False)
+    async def now_playing(command: Command, channel: discord.TextChannel, player_config: PlayerConfig):
+        '''Shows song currently playing'''
+        if not player_config.player.now_playing_item:
+            await channel.send('Nothing playing. Add a song with !play', delete_after=30)
+            return
+        title = player_config.player.now_playing_item.title
```

Comment:
Should just use a walrus operator with the `if` to the the item and then use the title down here, imo.

Reply:
was trying to keep the outer path as the happy path